crates/deterministic-wasi-ctx/Cargo.toml

```diff
@@ -12,11 +12,11 @@ categories = ["wasm"]
 anyhow = "1.0.56"
 async-trait = "0.1.53"
 cap-primitives = "1.0.3"
-wasi-common = ">= 4.0.0, < 7.1.0"
+wasi-common = ">= 4.0.0, < 8.1.0"
```

Comment:
This new API that we're using wouldn't be compatible with `wasi-common` 4.0 right?

Reply:
That'll work. We do lose compatibility with older versions of Wasmtime by making this change. That's probably fine though given it's unlikely someone on an older version of this crate will need a change to the implementation.

Reply:
Updated.